app/src/components/common/TmPage.vue

```diff
@@ -1,101 +1,43 @@
 <template>
-  <div class="tm-page">
-    <TmPageHeader v-if="!hideHeader" :tabs="tabs">
-      <h2 v-if="title" slot="title">
-        {{ title }}
-      </h2>
-      <h3 v-if="subtitle" slot="subtitle">
-        {{ subtitle }}
-      </h3>
-      <slot slot="menu-body" name="menu-body">
-        <TmBalance v-if="session.signedIn" />
-      </slot>
-      <slot slot="header-buttons" name="header-buttons" />
-    </TmPageHeader>
-    <main
-      class="tm-page-main"
-      :class="{ 'dark-background': darkBackground && session.signedIn }"
-    >
-      <CardSignInRequired v-if="signInRequired && !session.signedIn" />
-      <template v-else-if="managed">
-        <TmDataConnecting v-if="!connected" />
-        <TmDataLoading v-else-if="loading" />
-        <TmDataError v-else-if="error" />
-        <slot v-else-if="dataEmpty" name="no-data">
-          <TmDataEmpty>
-            <template slot="title">
-              <slot name="title" />
-            </template>
-            <template slot="subtitle">
-              <slot name="subtitle" />
-            </template>
-          </TmDataEmpty>
-        </slot>
-        <slot v-else name="managed-body" />
-      </template>
-      <slot />
-    </main>
+  <div
+    class="page"
+    :class="{ 'dark-background': darkBackground && session.signedIn }"
+  >
+    <CardSignInRequired v-if="signInRequired && !session.signedIn" />
+    <TmDataConnecting v-if="!connected" />
```

Comment:
are we showing this at any point?
```suggestion
```

Reply:
nope. nice find. the logic for connected doesn't even exist - it should all be removed. i'll do that as well.

https://github.com/luniehq/lunie/blob/develop/app/src/vuex/getters.js#L3
https://github.com/luniehq/lunie/blob/develop/app/src/vuex/modules/connection.js#L19